frontend/src/app/spot/styles/sass/components/modal-overrides.sass

```diff
@@ -1,7 +1,10 @@
 // Some backend modals come wrapped once extra. Because of the way
 // augmented modals are loaded, this is not so easy to change.
 // TODO: When refactoring the modal service, this should be removed
-.op-modal > .form:only-child
+.spot-modal > .form:only-child
```

Comment:
I am not so happy about placing that inside spot, as we don't want legacy code here. Maybe @b12f has a strong opinion on where to put that.

Reply:
Maybe putting it in these `$block-overrides.sass` files is the way to go @HDinger. We could do the same for the `spot-input` where foundation is in the way.

Reply:
what is the resulting suggestion? I do not get it yet.